vcd/resource_vcd_provider_vdc.go

```diff
@@ -128,7 +128,7 @@ func resourceVcdProviderVdc() *schema.Resource {
 			},
 			"nsxt_manager_id": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
Not sure if needed at this point, but just noting that this maybe needs an entry at `skip-upgrade-tests.txt`?

Reply:
It should not be needed. What worked in previous version as "Required" will work in this version as "Optional"